<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(131, 145, 196);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="chooseNumber">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:8px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="fixLabel3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Номер колонки:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseRegion">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
border-radius:8px;
background-color:rgb(255, 255, 255);
}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateMetrics">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>340</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:rgb(216, 219, 219);
border-radius:8px;
}

QPushButton:hover { background-color: rgb(77, 94, 132)}</string>
    </property>
    <property name="text">
     <string>Посчитать метрики
 и нарисовать график</string>
    </property>
   </widget>
   <widget class="QLabel" name="fixLabel2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Название региона:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxValue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fileName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="invalidStrings">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="maxValueFix">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Макс:</string>
    </property>
   </widget>
   <widget class="QLabel" name="validStrings">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="mediana">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="graph">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>600</width>
      <height>471</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius:8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="minValueFix">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Мин:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>411</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(216, 219, 219);
border-radius:8px
}
QPushButton:hover { background-color: rgb(77, 94, 132) }</string>
    </property>
    <property name="text">
     <string>Загрузить данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorString">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>601</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="fixLabel4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>              Поле ошибок:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fixLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Название файла:</string>
    </property>
   </widget>
   <widget class="QLabel" name="medianaFix">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Медиана:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
background-color: rgb(255, 255, 255);
border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fixLabel2_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Количество валидных строк:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(216, 219, 219);
border-radius: 8px;
}

QPushButton:hover { background-color: rgb(77, 94, 132)  }</string>
    </property>
    <property name="text">
     <string>Выбрать таблицу</string>
    </property>
   </widget>
   <widget class="QLabel" name="fixLabel2_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 8px;
}</string>
    </property>
    <property name="text">
     <string>  Количество удаленных строк:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:700; font-style:italic; text-decoration: underline; color:#5d5de2;&quot;&gt;Инструкция для пользователя &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5d5de2;&quot;&gt;1) Нажмите &amp;quot;Выбрать таблицу&amp;quot;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5d5de2;&quot;&gt;2) Выбирите регион&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5d5de2;&quot;&gt;3) Нажать &amp;quot;Загрузить данные&amp;quot;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5d5de2;&quot;&gt;4) Выбирите номер колонки&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5d5de2;&quot;&gt;5) Нажмите &amp;quot;Посчитать метрики и нарисовать график&amp;quot;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5d5de2;&quot;&gt;Вы можете выбрать другую колонку, но не регион.Для выбора другого региона перезапустите программу)))&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
